pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -35,9 +35,8 @@ assert !(openMPISupport && cudaSupport) || matchesCudatoolkit openmpi;
 assert !cudaSupport || matchesCudatoolkit magma;
 
 # confirm that mkl is sync'd across dependencies
-assert !mklSupport || mkl != null;
 assert !(mklSupport && cudaSupport) || matchesMkl magma;
-assert !mklSupport || (numpy.blasImplementation == "mkl" && numpy.blas == mkl);
+assert (mklSupport && mkl != null) || openblas != null;
```

Comment:
Why have these assertions changed? I ran into a hairy situation trying to use this with a `numpy` with `mkl`.

Reply:
It seems to be part of an overarching in-progress change: #74894